config/software/metasploit-framework.rb

```diff
@@ -2,6 +2,9 @@
 if linux? && File.exist?("/metasploit-framework")
   # supply current version of metasploit-framework at root of filesystem
   source path: "/metasploit-framework"
+elsif windows? && File.exist?('../metasploit-framework')
```

Comment:
This isn't an issue for CI; but I think this will need to be tightened up a bit; as it would be break in this scenario:

```
C:/metasploit-framework (clone check out)
C:/metasploit-omnibus
```

Because the `C:/metasploit-framework` directory will end up being the output directory for omnibus, it'll cause issues. On Github actions it's not an issue since everything builds under a temporary directory:

```
C:/a/metasploit-omnibus # The temp workspace
C:/a/metasploit-omnibus/metasploit-omnibus   # cloned via github action steps
C:/a/metasploit-omnibus/metasploit-framework # cloned via github action steps
```

Reply:
While likely worth adding a validation test to ensure this path is not `c:/metasploit-framwork`, I believe current automation in jenkins is similar to github actions and uses a path within the executor's home directory for checkout and build.

Existing usage is something like:
```
C:/Users/vagrant/metasploit-omnibus
C:/Users/vagrant/metasploit-framework
```